Driver/Driver/Custom/CustomXibView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CustomXibView.swift
+//  YonetimSistemi
+//
+//  Created by Fatih Doğan on 14.02.2023.
+//
+
+import UIKit
+
+class CustomXibView: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    private func commonInit (){
+        let view = loadViewFromNib()
+        view.frame = self.bounds
+        view.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.backgroundColor = UIColor.clear
+        addSubview(view)
+        clipsToBounds = true
+        
+    }
+    private func loadViewFromNib() -> UIView{
```

Comment:
defining with extra spacing is better.
 `private func loadViewFromNib() -> UIView { `